src/proxy-ui-api/frontend/src/store/modules/services.ts

```diff
@@ -1,12 +1,12 @@
 import { ActionTree, GetterTree, Module, MutationTree } from 'vuex';
 import { RootState } from '../types';
-import {Endpoint, Service, ServiceClient, Subject} from '@/types';
+import {Endpoint, Service, ServiceClient} from '@/types';
```

Comment:
We don't yet have a shared prettier setup, but before that, could these imports be adjusted in the ide to have spaces like { RootState }  ?

Reply:
fixed